Adding slides of the recent NEST and MESSAGEix-Nexus implementation

adrivinca opened the pull request:
Adding the material presented in two recent workshops and concerning NEST and the latest implementation of the MESSAGEix-Nexus model

# How to review:
compile and see if the _messageix-workshop_ looks fine and you can download the slides

(PS, I cannot assign reviewers)

adrivinca committed:
Adding slides of the recent NEST and MESSAGEix-Nexus implementation

danielhuppmann commented on message_ix_workshop.rst:
```diff
@@ -15,7 +15,8 @@ formulation and much more, please see the `documentation <https://docs.messageix
 Recent workshops
 """"""""""""""""
-
+* NEST and MESSAGE-ix Nexus module. September 2022, LUMS IAMs workshop and 
```
Seems that we are using a non-hyphen spelling for MESSAGEix
```suggestion
* NEST and MESSAGEix Nexus module. September 2022, LUMS IAMs workshop and 
```

danielhuppmann reviewed (commented):
I don't think that this NEST presentation makes a lot of sense in the list of MESSAGEix workshops. Instead, I suggest to add a new page for NEST in the "Talks & presentations" section of the page.

adrivinca committed:
Revert "Adding slides of the recent NEST and MESSAGEix-Nexus implementation"

This reverts commit cf15e83bdbce8a15694f685d9c8aa03840665e90.

adrivinca committed:
Adding a NEST page under the talks and presentations session

adrivinca committed:
non-hyphen spelling of MESSAGEix

adrivinca commented:
thanks @danielhuppmann.
I moved the content to a new page on NEST, also adding two other references. And fixed the MESSAGEix name

volker-krey commented on index.rst:
```diff
@@ -97,6 +97,8 @@ such as those given as part of summer schools or individual lectures.
 
    tutorials/pyam
    tutorials/eeg-coding-culture
+   tutorials/NEST
```
Would suggest to move this above 'eeg-coding-culture'.

adrivinca replied:
moved up

volker-krey commented on tutorials/NEST.rst:
```diff
@@ -0,0 +1,14 @@
+
+Presentations:
+
+- NEST and MESSAGEix Nexus module. September 2022, LUMS IAMs workshop and 
```
Adding a couple of introductory sentences about NEST and its relationship to MESSAGEix, also cross-referencing the MESSAGEix workshops, would make sense from my point of view.

adrivinca replied:
added som paragraph of description and cross-reference to the workshop page

volker-krey commented on tutorials/NEST.rst:
```diff
@@ -0,0 +1,14 @@
+
+Presentations:
+
+- NEST and MESSAGEix Nexus module. September 2022, LUMS IAMs workshop and 
```
Spell out LUMS and its location which not everyone will know.

adrivinca replied:
done

volker-krey commented on tutorials/NEST.rst:
```diff
@@ -0,0 +1,14 @@
+Presentations:
+
+- NEST and MESSAGEix Nexus module. September 2022, LUMS IAMs workshop and 
+  October 2022 Engage Stakeholder workshop (`download <https://github.com/adrivinca/teaching_material/raw/main/MESSAGEix-workshops/NEST_presentation_LUMS23_Sept_2022.pptx>`_)
```
For consistency with other material, I would suggest to deposit the presentation in PURE and link to that presentation instead of the one in your private GitHub repo.

adrivinca replied:
now substituted with a Pure link

volker-krey commented:
Thanks @adrivinca, I added a few specific comments to the committed files, suggesting a few smaller changes.

adrivinca commented:
Thanks @volker-krey . I cannot see the comments, maybe you have to post them as if you were "finalizing the review"

danielhuppmann commented:
@volker-krey, do you still want to make some comments on this PR?

adrivinca committed:
Edits requested from VK in PR #27

change order of talks and presentation
add more text on NEST and MESSAGEix-Nexus
add link to slides on Pure

adrivinca committed:
Updating index after rebasing to include the NEST page

adrivinca commented:
Thanks @volker-krey for the comments.
I addressed all the points and rebased after November edits. Hope it is good to go now.

danielhuppmann reviewed (approved):
Thanks @adrivinca!
